### DISCUSSION TITLE
How to make bucket replication exclude files to replicate

### DISCUSSION THREAD

**Discussion by creeew** [minio/minio#15935] [Q&A] [answered]
I found bucket replication have a prefix feature. File can be replicated if the name match the prefix.
But I need exclude some file when replicating. Please tell me how to exclude files to replicate.

**Comment by harshavardhana** [comment, marked as answer]
Just turn off versioning on them by using `mc version enable --help`

**Reply by klauspost** [reply]
? Maybe I misunderstand the question, but that isn't how I read the question.

Either way, there is no way to exclude files beside prefixes.

I suggest you re-arrange your files.

**Reply by creeew** [reply]
Thank you for your reply.
I can set prefix in bucket replication and make files replicate to another bucker if the file's name matched the prefix.
Also I try the mc version --excluded-prefixes, files cannot be replicated if the file's name matched the excluded-prefixes. 
It works, but the files that matched excluded-prefixes are unversioning which it is not sutiable for me.
If there is a excluded-prefixes just like prefix in bucket replication setting that would be so great. Maybe I should open a issue for that?

**Reply by harshavardhana** [reply]
You can't specifically skip files

**Reply by creeew** [reply]
> You can't specifically skip files

Why? Please tell me the reason. mc verison --excluded-prefixes already skipped files. And you mean we cannot skip file if the file was versioning?
